signing_service/signing_service/tests/tests.py

```diff
@@ -108,6 +114,7 @@ def setUp(self):
             '127.0.0.1',
             8000,
             2,
+            b'\xf3\x97\x19\xcdX\xda\x86tiP\x1c&\xd39M\x9e\xa4\xddb\x89\xb5,]O\xd5cR\x84\xb85\xed\xc9\xa17e,\xb2s\xeb\n1\xcaN.l\xba\xc3\xb7\xc2\xba\xff\xabN\xde\xb3\x0b\xa6l\xbf6o\x81\xe0;',
```

Comment:
I always recommend to add assignment to parameters in function when you are calling some function because then it is always obvious what parameters are you calling in specific function, but this is only my opinion :)

Reply:
This is good idea for adding as proposal to our coding style. Please note that if you use ie. `Pycharm` you can display function signature using `ctrl + move over`.

Reply:
Sure, but it is simpler to see it without going inside function :) I'm going to propose in ours coding style wiki.

Reply:
I agree, now we are sometimes adding it, sometimes not. We should be consistent about it.